### ISSUE TITLE
White-space pollution in table 3-04

### ISSUE THREAD

**Issue by manfredsc** [wmo-im/wmds#453]
### Details

Some entries in Table 3-04 have an unneeded space added at the end.
Search for ` ,"` in 3-04.csv resp. ` "@` in 3-04.ttl
These white-space characters are unexpected when parsing the table.

BTW: with a cursory glance at other tables, it seems also several other tables are affected.
Perhaps a project-wide
`for f in *.ttl; do sed -i 's/ \+"@/"@/g' "$f"; done`
would be in order.

### Requestor

Manfred Schwarb, schwarb@meteodat.ch

**Comment by joergklausen** [comment]
@amilan17 @marqh  Can you work together to fix this, please?

**Comment by manfredsc** [comment]
[ BTW: As a belated explanation for rising this issue, I did OSCAR requests and got unexpected whitespace pollution in the json output ]

I downloaded the pre-release and briefly inspected the files, looks good, I did not find any trailing whitespace before comma anymore. To be nitpicky, there are still a few whitespace characters at line endings (in 1-01-01.csv, 4-02.csv, 6-03.csv).

I saw that you changed a few "..." and "-"  signs, too. Just noted that some files have UTF-8 encodings and/or CRLF line terminators. The tables should perhaps be encoded in ASCII (CCITT-IA5 in WMO speak).

Thanks!
